.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,36 +1,30 @@
 ---
-name: "Bug report \U0001F41B"
-about: Create a report to help us improve
+name: "Bug Report \U0001F41B"
+about: Report a bug to help us to fix and improve the app
 labels: Bug
 ---
 
+### Describe the Bug
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+<!-- A clear and concise description of what the bug is. -->
+
+### Steps to Reproduce
+
+<!-- Describe the specific steps on how to reproduce the issue. -->
 
-**To Reproduce**
-Steps to reproduce the behavior:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+### Expected Behavior
+
+<!-- A clear and concise description of what you expected to happen. -->
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+### Related Issues
```

Comment:
was this ever needed?

Reply:
I think for things like duplicates, blocking, related this is helpful. For example for a new feature that needs work done in native and backend and thats too big for one issue.